apps/public/src/app/(general)/contact/Contact.tsx

```diff
@@ -4,13 +4,15 @@ import { Alert, Heading, Label, Paragraph, TextInput } from '@amsterdam/design-s
 import { Grid, SubmitButton } from '@meldingen/ui'
 import { useActionState } from 'react'
 
```

Comment:
We didn't set a test coverage limit, but `branches` of this component are at 77% since the airadescribedby is not tested.